backend/src/persistance/bo_descriptors.py

```diff
@@ -71,10 +88,53 @@ def validate(self, value):
         return isinstance(value, datetime)
 
 
-class BODate(PersistantAttr):
+class BODate(_PersistantAttr):
     @classmethod
     def data_type(cls):
         return date
 
     def validate(self, value):
         return isinstance(value, date)
+
+
+class BODict(_PersistantAttr):
+    @classmethod
+    def data_type(cls):
+        return dict
+
+    def validate(self, value):
+        try:
+            json.dumps(value, separators=(",", ":"))
```

Comment:
Should also validate that the value parameter is of type dictionary

Reply:
👍